Kitodo/pom.xml

```diff
@@ -97,7 +97,7 @@
         <dependency>
             <groupId>commons-cli</groupId>
             <artifactId>commons-cli</artifactId>
-            <version>1.2</version>
+            <version>1.4</version>
```

Comment:
`commons-cli` in version 1.2 is still used in  `OpacPica-Plugin`. So if you did not change it there, this library version is still deployed.

Reply:
I don't see this one in pom of OpacPica-Plugin.